web_src/less/_base.less

```diff
@@ -110,6 +110,8 @@
   --color-info-border: #a9d5de;
   --color-info-bg: #f8ffff;
   --color-info-text: #276f86;
+  --color-pull-request-list-branch-bg: #e8e8e8;
+  --color-pull-request-arrow-svg: #00000057;
```

Comment:
Not sure why we need these new colors, using `--color-secondary` and `--color-caret` should be consistent with arc-green theme.

Reply:
Now using --color-secondary and --color-secondary-dark-3 in both templates